modules/test/playwright/env/common.sh

```diff
@@ -85,7 +85,13 @@ function deploy_client_extensions {
 
 		for client_extension_name in $(cat ${client_extensions_list_file})
 		do
-			local client_extension_dir=$(find ${_PORTAL_PROJECT_DIR}/workspaces -type d -name "${client_extension_name}" | grep -v .releng | grep -v .npmscripts | grep -v node_modules)
+			if [[ "$client_extension_name" == *\/* ]]; then
+				workspace_path="/${client_extension_name%/*}"
+
+				client_extension_name="${client_extension_name##*/}"
+			fi
```

Comment:
Ooh that's some advanced bash regex flex :muscle: 

Looks good to me!

@izaera could there be any issues here?

Reply:
@thiago-buarqque @markocikos 

1. Frontend Infra is not the owner of the PW local setup script, it's "maintained by everybody", so I guess we are not the ones to formally aprove or reject these changes.
2. To the best of my knowledge, if you change this script, things are going to fail at CI because the CI has a different script to test things. You can contact the CI team to discuss about this change.
3. I don't know what's the current status of the local setup script, but my personal opinion is that it should mimick what the CI scripts do. In other words, the CI scripts should be the source of truth for how things should be deployed and tested.

Reply:
I see. Thanks @izaera. I will contact the CI team